Documentation/observability/metrics.rst

```diff
@@ -426,9 +426,19 @@ Name                                     Labels
 ======================================== ================================================== ========== ========================================================
 ``controllers_runs_total``               ``status``                                         Enabled    Number of times that a controller process was run
 ``controllers_runs_duration_seconds``    ``status``                                         Enabled    Duration in seconds of the controller process
+``controllers_group_runs_total``         ``status`` ``group_name``                          Enabled    Number of times that a controller process was run, labeled by controller group name (configurable)
```

Comment:
Can you add a comma between the labels and remove the `(configurable)` from the Description? The text you added below the table explains that it is configurable already. [1]

```suggestion
``controllers_group_runs_total``         ``status``, ``group_name``                          Enabled    Number of times that a controller process was run, labeled by controller group name
```